File: pages/page.js
var stdin= process.openStdin();
stdin.addListener("data", function(d)
{
var x= d.toString().trim().split("\n");
let b= Number(x[0]);
let sub = JSON.parse(x[1]);
let k= Number(x[2]);
var res= minimum_pages(sub,b,k);
    console.log(res);
});




function minimum_pages(arr,n,k)
{
    var allocated_value=-1;
    //Write your code here without removing the existing code
	//the variable 'arr' contains array of integers of size n.
	//modified the variable 'allocated_value' contain the output of the program in integer format.
    return allocated_value;
}

File: pages/page.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int k = sc.nextInt(); 
            int ans=g.minimum_pages(arr,n,k);  
            System.out.println(ans);
          
	}
}
class Solution
{
	public static int minimum_pages(int arr[],int n,int k)
        {
        
			int allocated_value = -1;
			//Write your code here without removing the existing code
			//the variable 'arr' contains array of integers of size n and k is an integer.
			//modified the variable 'allocated_value' contain the output of the program in integer format.
		    return allocated_value;
        }
}